todo_app.sql

```diff
@@ -0,0 +1,82 @@
+DROP TABLE IF EXISTS tasks;
+
+CREATE TABLE tasks
+(
+    id SERIAL NOT NULL PRIMARY KEY,
+    title VARCHAR(255) NOT NULL,
+    description TEXT,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP,
+    completed boolean NOT NULL DEFAULT 'FALSE'
+);
+
+ALTER TABLE tasks DROP COLUMN completed;
+
+ALTER TABLE tasks ADD COLUMN completed_at TIMESTAMP DEFAULT NULL;
+
+ALTER TABLE tasks ALTER COLUMN updated_at
+SET
+NOT NULL;
+
+ALTER TABLE tasks ALTER COLUMN updated_at
+SET
+DEFAULT CURRENT_TIMESTAMP;
+
+INSERT INTO tasks
+    (title, description, created_at, updated_at, completed_at)
```

Comment:
Figure out who to insert without defining the columns in this statement.

Reply:
I pushed a new commit that inserts the record without using column names.